UI/Admin.html

```diff
@@ -0,0 +1,139 @@
+						<div class="nav-left">
+							<a href="./frontPage.html"><h2><img class="laravel" src="./assets/images/logo.gif" height="40px" width="40px" alt="">  POLITICO</h2></a>
+						</div>
+		
+						<div class="nav-right">
+											
+						<ul class="header">
+								<li class="login-btn">
+									<a href="./allCandidates.html">Vote</a> 
+								</li>
+								<li class="signup-btn"> 
+								   <a href="./Application.html">Apply</a> 
+								</li>
+								<li>
+										<a href="./signin.html">
+											<button class="nav-button">Logout</button>
+										</a>
+								</li>
+							</ul>
+		
+							
+						</div>
+					</div>
+		
+				</nav>
+
+		<div class="edit-area" id="one">
+			<div class="container">
+				<h3>Admin Section</h3>
+				<a href="addParty.html" class="add-btn">
+						<button>
+								+
+							</button>
+				</a>
+
+				<div class="edit-card">
+					<h2 class="edit-card-title">
+						Action Congress Party
+					</h2>
+					<span class="edit-card-questions">
+						DATE CREATED  <b>12th, Dec 2018</b>
+					</span><br>
+					<div class="edit-card-button-area">
+						<button id="delete" class="one">
+							Delete
+						</button>
+
+						<button id="editButton">
+							Edit 
+						</button>
+					</div>
+					<div class="edit-card-footer" >
+						<small>created</small>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div class="edit-area" id="two">
+			<div class="container">
+				<div class="edit-card">
+					<h2 class="edit-card-title">
+						People Honor Congress
+					</h2>
+					<span class="edit-card-questions">
+						DATE CREATED  <b>12th, Dec 2018</b>
+					</span><BR>
+					<div class="edit-card-button-area">
+						<button id="delete" class="two">
+							Delete
+						</button>
+
+						<button id="editButton">
+							Edit 
+						</button>
+					</div>
+					<div class="edit-card-footer">
+						<small>created</small>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div class="edit-area" id="three">
+			<div class="container">
+				<div class="edit-card">
+					<h2 class="edit-card-title">
+						House Congress Party
+					</h2>
+					<span class="edit-card-questions">
+						DATE CREATED <b>12th, Dec 2018</b>
+					</span><br>
+					<div class="edit-card-button-area">
+						<button id="delete" class="three">
+							Delete 
+						</button>
+						<button id="editButton">
+							Edit 
+						</button>
+					</div>
+					<div class="edit-card-footer">
+						<small>created</small>
+					</div>
+				</div>
+			</div>
+		</div>
+
+<!-- 		<div id="deleteMeetupModal" class="delete-meetup-modal">
+			<div class="delete-meetup-modal-content">
+				<h5 class="meetup-message">Are you sure you want to delete meetup</h5>
+				<button id="closeModal">
+					No
+				</button>
+				<button>
+					Yes
+				</button>
+			</div>
+		</div> -->
+		
+	 	<script src="./assets/js/script.js"></script> 
+		<footer class="politico-footer">
+			<p>politico &copy; 2018</p>
+		</footer>
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well and also delete commented code blocks. Thanks